server/build_event_protocol/event_parser/event_parser.go

```diff
@@ -32,27 +32,57 @@ const (
 	workflowConfiguredPriority = 5
 )
 
-func parseEnv(commandLine *command_line.CommandLine) map[string]string {
-	envVarMap := make(map[string]string)
+var (
+	optionsToParse = []string{
```

Comment:
nit: you can turn this into a map for lookups (though in practice the list is fine too)

	optionsToParse = map[string]bool{
		"remote_cache": true,
		...
	}

Reply:
Done.